src/validation.cpp

```diff
@@ -2240,12 +2249,16 @@ bool RewardInvites(
         return false;
     }
 
-    const bool pog2 = force_pog2 || height >= params.pog2_blockheight;
-    assert(!pog2 || cgs_selector);
+    size_t total_winners = 0;
 
-    const auto total_winners = pog2 ? 
-        pog2::ComputeTotalInviteLotteryWinners(lottery_params, params) :
-        pog::ComputeTotalInviteLotteryWinners(height, lottery_params, params);
+    const bool pog2 = height >= params.pog2_blockheight;
+    if(force_pog2 || height >= params.imp_fix_invites_blockheight) {
```

Comment:
why `force_pog2` here?

Reply:
there is a new simulatelottery RPC call and we force enable this during simulation